kafka-connect-upload-source/src/main/java/org/radarbase/connect/upload/converter/ConverterLogRepository.kt

```diff
@@ -45,33 +44,38 @@ interface LogRepository {
 }
 
 class ConverterLogRepository(
-        val uploadClient: UploadBackendClient): LogRepository {
+        val uploadClient: UploadBackendClient) : LogRepository {
     private val logContainer = mutableMapOf<Long, MutableList<Log>>().withDefault { mutableListOf() }
 
+    private fun get(recordId: Long): MutableList<Log> =
+            logContainer.getOrPut(recordId, { mutableListOf() })
```

Comment:
Is this going to be called from multiple threads (e.g. multiple tasks or multithreaded converters?) In that case, the logContainer should probably be replaced by a `ConcurrentHashMap`, and the list replaced by a `ConcurrentLinkedQueue`. See #60.